doc/releases/changelog-dev.md

```diff
@@ -96,27 +95,39 @@
 
   ```
 
-<h4>Make use of more methods to map from molecules 🗺️</h4>
-
+<h4>More methods to map molecules to qubits 🗺️</h4>
 
-* Added new function `qml.bravyi_kitaev` to map fermionic Hamiltonians to qubit Hamiltonians.
+* A new function called `qml.fermi.bravyi_kitaev` has been added to map fermionic 
+  Hamiltonians to qubit Hamiltonians.
   [(#5390)](https://github.com/PennyLaneAI/pennylane/pull/5390)
 
-  ```python
-  import pennylane as qml
-  fermi_ham = qml.fermi.from_string('0+ 1-')
-
-  qubit_ham = qml.bravyi_kitaev(fermi_ham, n=6)
-  ```
+  In addition to `qml.fermi.jordan_wigner` and `qml.fermi.parity_transform`, the
+  Bravyi-Kitaev mapping is available via `qml.fermi.bravyi_kitaev` to map
+  fermionic Hamiltonians to qubit Hamiltonians. Simply provide a fermionic 
+  Hamiltonian (created from `from_string`, `FermiA`, `FermiC`, `FermiSentence`, 
+  or `FermiWord`) and the number of qubits / spin orbitals in the system, `n`:
 
   ```pycon
+  >>> fermi_ham = qml.fermi.from_string('0+ 1-')
+  >>> qubit_ham = qml.bravyi_kitaev(fermi_ham, n=6)
   >>> print(qubit_ham)
   -0.25j * Y(0.0) + (-0.25+0j) * X(0) @ Z(1.0) + (0.25+0j) * X(0.0) + 0.25j * Y(0) @ Z(1.0)
```

Comment:
Would update these lines:
```
>>> qml.bravyi_kitaev(fermi_ham, n=6)
(
    -0.25j * Y(0)
  + (-0.25+0j) * (X(0) @ Z(1))
  + (0.25+0j) * X(0)
  + 0.25j * (Y(0) @ Z(1))
)
```
@soranjh @ddhawan11 would it be challenging to cast the coeffs to real? Do we ever expect them to be complex?

Reply:
@trbromley In this simple example, `0.25j` is a large number to neglect and cast to real. But we can give you a bit larger examples where the coeffs have negligible imaginary part. Is that what you want?